### ISSUE TITLE
Add a function to send messages to MessageLog

### ISSUE THREAD

**Issue by PiotrRajtar** [cubewise-code/tm1py#621]
It would be useful for tm1py to allow sending messages to the MessageLog, so that they can be easily browsed and picked up by Pulse alerts.

Currently I am doing something similar using an unbound TI, but is not ideal as whitespaces have to be replaced with some other separator, e.g. underscore, otherwise they get removed:
```
                logging.exception(f"Unable to send values to cube [{log_cube}].")
                self.tm1.processes.execute_ti_code(f"LogOutput('ERROR', 'TM1py_[{path.basename(__file__)}]"
                                                   f"_{type(ex).__name__}__Please_check_[{LOG_FILE}]_for_details.');")
```
Ideally, the function should be supported as a custom logging handler in the logging module.

**Comment by zsoltmoravcsik** [comment]
+1 That would be a nice feature, could we maybe leverage the REST API log
related endpoints? Maybe just give a simple feature to post via rest API a
message log entry like Logoutput in TIs. It would help
REST-based application transparency and collect logs in one place which
make log tracing simpler for solutions like Pulse or other ELK stack
solutions.

**Comment by rkvinoth** [comment]
@PiotrRajtar I don't think there is a way to add `LOGOUTPUT` via REST API (I could be wrong).

But the unboundTI way works. The issue with your code is that, `execute_ti_code` method expects an `iterable` but you are passing a `str`. It is trying to convert your `str` into an `iterable` and that is why the whitespaces are getting removed.
This should work:

```python
try:
    pass
except Exception as ex:
    logging.exception(f"Unable to send values to cube [{log_cube}].")
    self.tm1.processes.execute_ti_code(lines_prolog=[f"LogOutput('ERROR', 'TM1py [{path.basename(__file__)}] "
                                                     f"{type(ex).__name__} please check [{LOG_FILE}] for details.');"])

```
I highly recommend the usage of parameter names (`lines_prolog`) in the code (though it is the only mandatory parameter), as it makes the code more readable.

@MariusWirtz Would it be possible to build a custom method that would take the `loglevel` and `message` then write it to the serverlog using TI (we can have the `require_admin` decorator)?

**Comment by MariusWirtz** [comment]
Good point @rkvinoth. The function expects an iterable. I oversaw this.
AFAIK we have to use unbound ti processes for writing to the message log.

 @rkvinoth please feel free to open a Pull Request!
